parts/linux/cloud-init/artifacts/ensure-no-dup.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Add dudup ebtable rules for promisc mode
+After=kubelet.service
+[Service]
+Restart=on-failure
+RestartSec=30
```

Comment:
30s sees too large, it's very likely cx can observe the duplicated packet issue.

I'd say 3s or even 1s? If the cbr0 is not there it does no harm to check and return . once it's configured it's done

Reply:
Yep, let's try 1s. 
Again this kinda comes back to the issue that this really isn't the most elegant way of doing this. Kubelet's dockershim has full visibility into pod lifecycles so it's able to do this check everytime it spins up a podsandbox whereas here we're at the mercy of good timing. But yea, reducing it to 1s should make this work better.